docs/advanced-config.md

```diff
@@ -3,29 +3,25 @@
 
 ## Register a Github Oauth App
-Information is available on the [Github Developer Documentation](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/). Fill out the fields however you like, however **it is critical that your _authorization callback URL_ is correct**. This tells Github where to send a user after they've authenticated, and should match the URL of your Oauth provider. Use `https://your.app.com/callback` with Vencax's Oauth provider explained below.
+Information is available on the [Github Developer Documentation](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/). Fill out the fields however you like, however **it is critical that your _authorization callback URL_ is correct**. This tells Github where to send a user after they've authenticated, and should match the callback URL of your Oauth server. A good bet is `https://your-auth-server.com/callback`, but check what URL your server is listening for in the sourcecode.
 
-After clicking "Register Application", you'll be provided with a `client_id` and `client_secret` on the next page that you'll use when configuring your Oauth provider
+After clicking "Register Application", you'll be provided with a `client_id` and `client_secret` on the next page that you'll use when configuring your Oauth Server
 
-## Configure an External Oauth Provider
+## Configure an External Oauth Server
+Netlify CMS is meant to be an open platform, so we're always looking to expand the ecosystem and find new ways to use it. Below is a list of currently submitted Oauth providers, but again feel free to [submit a pull request](https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md) if you think this list is missing anything.
 
-We'll be using Vencax's [Netlify CMS Github Oauth Provider](https://github.com/vencax/netlify-cms-github-oauth-provider) in this example. This repository is convenient because it mirrors Netlify's own Github Oauth implementation, and can support repositories from Github.com or Enterprise accounts.
+### External Oauth Servers:
+| Author     | Supported Git hosts | Language | Link |
+|------------|---------------------|----------|------|
+| Vencax     | Github & GH Enterprise | Node.js  | [Repo](https://github.com/vencax/netlify-cms-github-oauth-provider) |
 
-Configuration is accomplished with environment variables, and is probably easiest with a `.env` file. An example would be:
-
-```
-NODE_ENV=production
-OAUTH_CLIENT_ID=f432a9casdff1e4b79c57
-OAUTH_CLIENT_SECRET=pampadympapampadympapampadympa
-GIT_HOSTNAME=https://github.some.domain.com
-```
-`GIT_HOSTNAME` is only necessary if you're using Github Enterprise. Additional config and install information for Vencax's Oauth provider is available in [its repository](https://github.com/vencax/netlify-cms-github-oauth-provider).
```

Comment:
I've submitted a pull request to Vencax's repo that includes the Github Enterprise functionality, as well as documentation for it. Considering that all of those environment variables are specific to his repo, I figured I would remove it. I could add it back in some form though potentially.

From my PR on Vencax's repo:
```md
### Auth Provider Config

Configuration is done with environment variables, which can be supplied as command line arguments, added in your app  hosting interface, or loaded from a .env ([dotenv](https://github.com/motdotla/dotenv)) file.

**Example .env file:**

```
NODE_ENV=production
OAUTH_CLIENT_ID=f432a9casdff1e4b79c57
OAUTH_CLIENT_SECRET=pampadympapampadympapampadympa
REDIRECT_URL=https://your.server.com/callback
GIT_HOSTNAME=https://github.website.com
```

**Client ID & Client Secret:**
After registering your Oauth app, you will be able to get your client id and client secret on the next page.

**Redirect URL (optional):**
Include this if you  need your callback to be different from what is supplied in your Oauth app configuration.

**Git Hostname (Optional):**
This is only necessary for use with Github Enterprise.
```